cluster/charts/crossplane-types/crds/apiextensions.crossplane.io_compositions.yaml

```diff
@@ -25,27 +25,21 @@ spec:
     status: {}
   validation:
     openAPIV3Schema:
-      description: Composition defines the group of resources to be created when a
-        compatible type is created with reference to the composition.
+      description: Composition defines the group of resources to be created when a compatible type is created with reference to the composition.
```

Comment:
These are non-functional changes that happen as part of CRD generation after bumping `crossplane-runtime` / `controller-runtime`. It is somewhat surprising to me that this would be modified without changing the `controller-tools` version though :thinking: